test/e2e/actions/deploy/deploy_operator.go

```diff
@@ -84,6 +85,9 @@ func CreateInitialDeployments(testData *model.TestDataProvider) {
 				deployment.Namespace = testData.Resources.Namespace
 				deployment.Spec.Project.Namespace = testData.Resources.Namespace
 			}
+			j, _ := json.MarshalIndent(deployment, "", " ")
+			fmt.Println("DEBUG INITIAL DEPLOYMENTS", string(j))
```

Comment:
this seems redundant for the final version

Reply:
Good point, removed